src/main/java/mate/academy/onlinebookstore01/OnlineBookStore01Application.java

```diff
@@ -9,5 +9,4 @@ public class OnlineBookStore01Application {
     public static void main(String[] args) {
         SpringApplication.run(OnlineBookStore01Application.class, args);
     }
```

Comment:
Please, add `CommandLineRunner` bean in accordance with the task requirements.
<img width="862" alt="image" src="https://github.com/DPotapov93/OnlineBookStore/assets/140437083/ed47bc2d-a985-4722-886f-82d0e4734371">